uikit/src/main/res/layout/activity_bank_points.xml

```diff
@@ -31,6 +31,17 @@
             android:background="@color/dim_background"
             android:visibility="gone"/>
 
+        <com.midtrans.sdk.uikit.widgets.FancyButton
+            android:id="@+id/button_pay_without_point"
+            android:layout_centerHorizontal="true"
+            app:fb_iconResource="@drawable/ic_pay_without_bank_point"
+            app:fb_text="@string/pay_without_bank_point"
```

Comment:
Need to use string formatter for this one.
![screenshot_2017-11-20-15-11-43-710_com midtrans sdk demo development](https://user-images.githubusercontent.com/30827802/33008592-48cf7fe6-ce06-11e7-8ebb-96ac1f6ffc83.png)